macros/add_property_labels.sql

```diff
@@ -0,0 +1,42 @@
+{% macro add_property_labels(passthrough_var_name, cte_name) %}
```

Comment:
When using the same variables you shared in your validation (database, schema, passthrough columns, etc) I noticed that I was never able to see label results. They are all null for me. Please view `joes_contact_validation` cell in your Hex validations notebook for an example.

Do you know why that may be?

Reply:
I actually was able to expand my validations to do this for `company`, `deal`, and `ticket` as well. It looks like I am seeing the same issue where no labels are being brought through in my end models. Was there by chance a change you had made following your validations notebook creation that would impact these results?

Reply:
It looks like following your changes I can see the expected results. Thanks!